reana_server/reana_admin/cli.py

```diff
@@ -723,9 +723,22 @@ def queue_consume(
 
 
 @reana_admin.command()
-def retention_rules_apply() -> None:
+@click.option(
+    "--force-date",
+    type=click.DateTime(),
+    help="Change the date and time used to decide which rules to apply.",
+)
+def retention_rules_apply(force_date) -> None:
```

Comment:
Yes, I think it would be good to ask for confirmation all the time, since this is really a potentially dangerous admin operation.

Instead of warning with the current message:

> You want to consider **ALL** the workflows while forcing the date to be 2029-01-12 00:00:00. Are you sure? [y/N]: N

we could print something like:

    Going to delete non-retained workspace files for ALL users and ALL their workflows as if it were 2029-01-12 00:00:00.

    Going to delete non-retained workspace files for JOHN.DOE@EXAMPLE.ORG and ALL their workflows as if it were 2029-01-12 00:00:00.

    Going to delete non-retained workspace files for 111-222-333 workflow as if it were 2029-01-12 00:00:00.

    Are you sure you want to continue? (y/N)

Reply:
BTW for extra safety we could even allow providing both `-id` and `-w` options at the same time.

Currently this is not possible:

```console
$ kubectl exec -i -t deployment/reana-server -c rest-api -- flask reana-admin retention-rules-apply --force-date 2029-01-12 -w ec629a6b-66ae-405a-8de0-f68f6b0662c3 --id 00000000-0000-0000-0000-000000000000
Cannot provide --email/--id and --workflow at the same time.
```